src/Prefixer.js

```diff
@@ -63,18 +63,19 @@ export function prefix (value, length) {
 		case 8116: case 7059: case 5753: case 5535:
 		case 5445: case 5701: case 4933: case 4677:
 		case 5533: case 5789: case 5021: case 4765:
+			var temp = value
 			// stretch, max-content, min-content, fill-available
 			if (strlen(value) - 1 - length > 6)
 				switch (charat(value, length + 1)) {
 					// (m)ax-content, (m)in-content
 					case 109:
 						return replace(value, /(.+:)(.+)-([^]+)/, '$1' + WEBKIT + '$2-$3' + '$1' + MOZ + '$2-$3') + value
-					// (f)ill-available
-					case 102:
-						return replace(value, /(.+:)(.+)-([^]+)/, '$1' + WEBKIT + '$2-$3' + '$1' + MOZ + '$3') + value
 					// (s)tretch
 					case 115:
-						return prefix(replace(value, 'stretch', 'fill-available'), length) + value
+						temp = (value = replace(value, 'stretch', 'fill-available')) + temp
```

Comment:
```js
return indexof(value, 'stretch') > -1 ? ... : value
```

Reply:
the indexof approach doesnt work because of this `fit-content`:
https://github.com/thysultan/stylis.js/blob/9799734dcf2b04c1216818f8b74cba92b6de4f76/test/Prefixer.js#L130

this is actually missing from the comment above `case 102:`, should be:
```
// (f)ill-available, (f)it-content
```

Reply:
This should work

```js
// (min|max)?(width|height|inline-size|block-size)
case 8116: case 7059: case 5753: case 5535:
case 5445: case 5701: case 4933: case 4677:
case 5533: case 5789: case 5021: case 4765:
	// stretch, max-content, min-content, fill-available
	if (strlen(value) - 1 - length > 6)
		switch (charat(value, length + 1)) {
			// (f)ill-available, (f)it-content
			case 102: length = charat(value, length + 3)
			// (m)ax-content, (m)in-content
			case 109:
				return replace(value, /(.+:)(.+)-([^]+)/, '$1' + WEBKIT + '$2-$3' + '$1' + MOZ + (length == 108 ? '$3' : '$2-$3')) + value
			// (s)tretch
			case 115:
				return ~indexof(value, 'stretch') ? prefix(replace(value, 'stretch', 'fill-available'), length) + value : value
		}
	break
```

Reply:
Ok, went with that one

Reply:
Did you see the `case 102: length = charat(value, length + 3)` amendment?

Reply:
Oh, no - I've focused on the `indexof` part in a hurry. This is supposed only to compress the code, right? Either way - I've pushed out this change right now.